docs/how-to-guides/manage-instances/launch-customized-virtual-machines-with-multipass-and-cloud-init.md

```diff
@@ -0,0 +1,102 @@
+(how-to-guides-manage-instances-launch-customized-virtual-machines-with-multipass-and-cloud-init)=
+# Launch customized virtual machines with Multipass and cloud-init
+
+If you want to set up a virtual machine with a specific environment or configuration, you can use the launch command along with a custom cloud-init YAML file and an optional post-launch health check to ensure everything is working correctly.
+
+Below are some common examples of using cloud-init YAML and Multipass commands to create a customized virtual machine setups. In each case, a pre-defined cloud-init YAML file is stored in the repository, but users are also free to create and user their own custom cloud-init configurations.
+
+## 📦 anbox-cloud-appliance
+Launch with:
+```{code-block} text
+multipass launch --name anbox-cloud-appliance --cpus 4 --memory 4G --disk 50G --timeout 900 --cloud-init https://raw.githubusercontent.com/canonical/multipass/refs/heads/cloud-init-yaml/data/cloud-init-yaml/cloud-init-anbox.yaml
```

Comment:
Can we break this up into several lines so I can see the whole command without side scrolling?